recipes/braggedgemodeling/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "braggedgemodeling" %}
+{% set version = "0.1.1a2" %}
+{% set git_rev = "v0.1.1a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ornlneutronimaging/{{ name }}/archive/{{ git_rev }}.tar.gz
+  sha256: 5380dffdd271cb725c7c244fb94b306e93cfe318c4e3d134774712cff5ed5b35
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.5
+    - pyyaml
+    - numpy
+    - scipy
+    - matplotlib
+    - diffpy.structure
+    - periodictable
+    
+test:
+  imports:
+    - bem
+
+about:
+  home: https://github.com/ornlneutronimaging/braggedgemodeling
+  license: MIT
+  license_family: MIT
+  # license_file: LICENSE
```

Comment:
Please open an issue upstream and, if possible, package the license manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually